config/maps.txt

```diff
@@ -13,12 +13,8 @@ Format:
 	votable (is this map votable)
 endmap
 
-map heliostation
-	minplayers 25
-	votable
-endmap
```

Comment:
why are you changing this config file

Reply:
I had previously commited everything needed to add a functioning heliostaton, and then commited it out so it wouldnt also straight up add it. And now that heliostation is there. Its still carrying over the removal of stuff in that file. Ill fix it when i get home.

Reply:
not a huge deal as its a config item but it just seemed a bit weird lol